MonoGame.Framework/MathHelper.cs

```diff
@@ -266,13 +266,13 @@ public static float SmoothStep(float value1, float value2, float amount)
         /// <param name="radians">The angle in radians.</param>
         /// <returns>The angle in degrees.</returns>
         /// <remarks>
-        /// This method uses double precission internally,
+        /// This method uses double precision internally,
```

Comment:
For backward compatibility, I think we should leave the orignal MathHelper as-is and that internal double precision.